protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -122,6 +122,31 @@ definitions:
           - SOURCE_RETRIEVAL_ERROR
           # Errors casting to appropriate type
           - DESTINATION_TYPECAST_ERROR
+  AirbyteFileTransferRecordMessage:
+    type: object
+    additionalProperties: true
+    required:
+      - stream
+      - file
+      - emitted_at
+    properties:
+      namespace:
+        description: "namespace the data is associated with"
+        type: string
+      stream:
+        description: "stream the data is associated with"
+        type: string
+      file:
+        description: "transfer file data"
+        type: object
+        existingJavaType: com.fasterxml.jackson.databind.JsonNode
```

Comment:
Could we have an actual object or add the file related entries to the Message

Reply:
Just added AirbyteFileType in definitions.